packages/commons/test/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "description": "This file is used to download npm dependencies for this project",
+  "repository": {},
+  "license": "MIT",
```

Comment:
To avoid confusion, this should probably be MPL-2.0 since it's under `/packages/commons`